### ISSUE TITLE
test registration date remains from deleted test after adding a new test

### ISSUE THREAD

**Issue by crcsn** [corona-warn-app/cwa-app-android#2005]
## Avoid duplicates
* [ ] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ ] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [ ] Bug is not already reported in another issue

## Technical details

- Device name: Samsung Galaxy S10
- Android version: 10
- App version: 1.10.1

## Steps to reproduce the issue

1. I added a test via QR Code on Dec 25. Test result page correctly showed "Registered on 12/25/20" (Test returned negative on Dec 26)
2. Today (Dec 29) I deleted the old test to add a new one.
3. Afterwards (Dec 29), I registered that new test via QR Code. The result page still shows "Registered on 12/25/20"

![tempFileForShare_20201229-103344](https://user-images.githubusercontent.com/19310481/103274276-90b18180-49c1-11eb-9432-e7a320684775.jpg)

## Expected behaviour

Test registration date should correctly be 29 Dec 2020, not the registration date of the last test.

## Additional context

---
Internal Tracking ID: [EXPOSUREAPP-4484](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4484)

**Comment by dsarkar** [comment]
Dear @crcsn,

Thank you for your report. We have created a ticket for the bug in the internal Jira system. Any further developments will be notified here in this Github issue.
  
Best wishes,
DS

---
Corona-Warn-App Open Source Team
Internal Tracking ID: [EXPOSUREAPP-4484](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4484)

**Comment by dsarkar** [comment]
Dear @crcsn, and community,

Feedback from developers:  
- confirmed the issue on 1.9.1 and 1.10.1
- always displays the date of the first scan
- the test result retrieved is the correct one (despite showing the wrong date). 
- It only seems to be a display issue of the date.

Best wishes,
DS

---
Corona-Warn-App Open Source Team

**Comment by PhilippNowak96** [comment]
Opened a PR to fix this: #2024

**Comment by dsarkar** [comment]
@PhilippNowak96 Devs are informed about PR. Thanks again. Best wishes, DS

---
Corona-Warn-App Open Source Team

**Comment by MikeMcC399** [comment]
"Reset the initial registration date on test removal correctly (EXPOSUREAPP-4484) (closes #2005)" https://github.com/corona-warn-app/cwa-app-android/pull/2024 is included in [v1.11.0-RC1](https://github.com/corona-warn-app/cwa-app-android/releases/tag/v1.11.0-RC1).

**Comment by MikeMcC399** [comment]
In CWA Android 1.11.0 I registered a dummy test yesterday, Jan 30, 2021

![Test yesterday](https://user-images.githubusercontent.com/66998419/106377623-9f170280-639e-11eb-8bb4-d909ecf81801.jpg)

then deleted the test today, Jan 31, 2021. Then I registered a new dummy test today.

![Test today](https://user-images.githubusercontent.com/66998419/106377638-bce46780-639e-11eb-908f-5433f2a82d5a.jpg)

The old date has been removed and replaced by the new date, which demonstrates that the issue has been resolved.

So it should now be possible to close this hot topic issue.
